Xamarin.PropertyEditing.Tests/MockControls/MockSampleControl.cs

```diff
@@ -11,11 +11,13 @@ public class MockSampleControl : MockControl
 	{
 		public MockSampleControl ()
 		{
+			AddProperty<TimeSpan> ("TimeSpan", ReadWrite, valueSources: ValueSources.Local | ValueSources.Resource | ValueSources.Binding);
```

Comment:
Not sure `TimeSpan` should support `ValueSources.Resource`  here...
<img width="546" alt="Screen Shot 2019-03-25 at 11 52 12" src="https://user-images.githubusercontent.com/271363/54917916-cffd9500-4ef4-11e9-88b0-03fe7dca3152.png">

Reply:
We might not have any example resources in the standalone, but there's nothing wrong with this.

Reply:
makes sense to include here a NSDatePicker with only time elements (like we have with Dates)

 this.datePicker = new NSDatePicker {
				ControlSize = NSControlSize.Small,
				DatePickerElements = NSDatePickerElementFlags.HourMinuteSecond,
				DatePickerStyle = NSDatePickerStyle.TextFieldAndStepper,
				Font = NSFont.SystemFontOfSize (NSFont.SystemFontSizeForControlSize (NSControlSize.Small)),
				TranslatesAutoresizingMaskIntoConstraints = false
			};